### DISCUSSION TITLE
https://rucsoundings.noaa.gov down

### DISCUSSION THREAD

**Discussion by aeharding** [NOAA-GSL/GSL-Regional-Model-Forum#48] [Q&A] [answered]
Redirects to https://gsl.noaa.gov/tmp/302-redirect

**Comment by Brian-Jamison** [comment, marked as answer]
Hi Alexander,

NOAA management has decreed that all of our web pages must be section 508
compliant (https://www.section508.gov/) or they would have to be shut down
to the public.  I'm currently working on reaching compliance.  I hope we
can have the page available again soon, but I don't have an ETA right now.

Also, GSL is a research laboratory, and the products that we provide should
not be used operationally, nor should they be used for operational decision
making, which is the main reason that the site has been removed from public
access.  It's possible that the site be restored if it's brought into
compliance, and includes a caveat that it's not to be used for operational
purposes.

The site https://weather.uwyo.edu/upperair/ might be helpful in the
meantime.  Also, twisterdata.com has some point & click sounding
capabilities.

Apologies for the interruption,

Brian

**Reply by aeharding** [reply]
Hi @Brian-Jamison, the site now says,

> On Monday, September 30, a number of legacy websites were permanently removed. These sites were no longer being maintained and did not meet security and design requirements mandated by NOAA. They were intended for research purposes and are not designed for operational use, such as for commercial purposes or the safety of life and property. There is no plan to restore any access to these sites; they have been permanently retired.

Is that true, or is there still a plan to make the site public eventually again?

Alternatively, is there a free/government public endpoint to receive a sounding for given coordinates?

Thanks
Alex
